src/generators/cfml.ts

```diff
@@ -0,0 +1,113 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + quote(request.url as string)  + '");\n';
+  cfmlCode += 'httpService.setCharset("utf-8");\n';
+
+  cfmlCode +=
+    'httpService.setMethod("' + quote(request.method.toUpperCase()) + '");\n';
```

Comment:
If you're passing it as a string, just pass it as is

```suggestion
    'httpService.setMethod("' + quote(request.method) + '");\n';
```

curl lets you do `curl -X PoSt` and it will actually send the letter case you specify.